bucket/juce.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://juce.com/",
+    "description": "JUCE is an open-source cross-platform C++ application framework. The leading framework for multi-platform audio applications",
+    "version": "5.4.7",
+    "license": {
+        "identifier": "JUCE Personal,JUCE Indie,JUCE Pro,JUCE Educational,ISC,GPL-3.0-or-later",
+        "url": "https://juce.com/juce-5-license"
+    },    
+    "url": "https://github.com/juce-framework/JUCE/releases/download/5.4.7/juce-5.4.7-windows.zip",
+    "extract_dir": "JUCE",
+    "bin": "Projucer.exe",
+    "shortcuts": [
+        [
+            "Projucer.exe",
+            "Projucer"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/juce-framework/JUCE"
+    },
+    "autoupdate": {
+        "url": "https://github.com/cmderdev/cmder/releases/download/$version/juce-$version-windows.zip"
```

Comment:
url is wrong

Reply:
Oh! I copied an example URL! Thank you! Will fix it!